wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController.swift

```diff
@@ -107,9 +108,7 @@ final class StartUIViewController: UIViewController {
     }
 
     var showsGroupSelector: Bool {
-        SearchGroup.all.count > 1 &&
-            userSession.selfUser.canSeeServices &&
-            userSession.defaultProtocol != .mls
+        isAppsTabVisible && SearchGroup.all.count > 1 && userSession.selfUser.canSeeServices
```

Comment:
question: is the logic still ok? or should we look directly to the feature config value?